optional_plugins/varianter_yaml_to_mux/tests/test_multiplex.py

```diff
@@ -10,17 +10,16 @@
 basedir = os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..')
 basedir = os.path.abspath(basedir)
 
-AVOCADO = os.environ.get("UNITTEST_AVOCADO_CMD", "./scripts/avocado")
-
-
-DEBUG_OUT = """Variant mint-debug-amd-virtio-07c6:    amd@selftests/.data/mux-environment.yaml, virtio@selftests/.data/mux-environment.yaml, mint@selftests/.data/mux-environment.yaml, debug@selftests/.data/mux-environment.yaml
-    /distro/mint:init         => systemv@selftests/.data/mux-environment.yaml:/distro/mint
-    /env/debug:opt_CFLAGS     => -O0 -g@selftests/.data/mux-environment.yaml:/env/debug
-    /hw/cpu/amd:cpu_CFLAGS    => -march=athlon64@selftests/.data/mux-environment.yaml:/hw/cpu/amd
-    /hw/cpu/amd:joinlist      => ['first_item']@selftests/.data/mux-selftest.yaml:/hw/cpu + ['second', 'third']@selftests/.data/mux-selftest.yaml:/hw/cpu/amd
-    /hw/disk/virtio:disk_type => virtio@selftests/.data/mux-environment.yaml:/hw/disk/virtio
-    /hw/disk:corruptlist      => nonlist@selftests/.data/mux-selftest.yaml:/hw/disk
-    /hw:corruptlist           => ['upper_node_list']@selftests/.data/mux-selftest.yaml:/hw
+AVOCADO = os.environ.get("UNITTEST_AVOCADO_CMD", "../scripts/avocado")
```

Comment:
This will break the usage of `UNITTEST_AVOCADO_CMD` because one hardcodes it to `./scripts/avocado` environment but here you require different level. I think better solution would be `os.chdir` at the beginning of each test (using AVOCADO).